### DISCUSSION TITLE
Github student pack

### DISCUSSION THREAD

**Discussion by Iris-Smok** [community/community#57460] [GitHub Education] [answered]
### Github student pack 

Question

### Body

Hi all, 

When you sign up for the Github student pack do you know for how long it's available? 

Thanks,
Iris

**Comment by samus-aran** [comment, marked as answer]
Hi @Iris-Smok 👋 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We usually ask to reverify between one to two years, but we may require this earlier when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.

Hope this helps 👍
